plugins/task-plugin/src/variable/project-path-variable-resolver.ts

```diff
@@ -9,17 +9,17 @@
  **********************************************************************/
 
 import { injectable } from 'inversify';
-import Uri from 'vscode-uri';
 import * as che from '@eclipse-che/plugin';
 import * as theia from '@theia/plugin';
 import * as startPoint from '../task-plugin-backend';
 
 const VARIABLE_NAME = 'current.project.path';
-const SELECTED_CONTEXT_COMMAND = 'theia.plugin.workspace.selectedContext';
 const PROJECTS_ROOT_VARIABLE = 'CHE_PROJECTS_ROOT';
 const ERROR_MESSAGE_TEMPLATE = 'Can not resolve \'current.project.path\' variable.';
 /**
  * Contributes the variable for getting path for current project as a relative path to the first directory under the root workspace.
+ * 
+ * @deprecated will be removed soon. Use ${workspaceFolder} macro instead.
```

Comment:
I think it's not enough just replace `${workspaceFolder}` by `${current.project.path}`. Instead user should use `${workspaceFolder}/theia` for example

Reply:
There will be created PR into docs that describes that fact, that this macro is deprecated and has to be replaced